org.vikdevelop.VCalculator.json

```
@@ -0,0 +1,36 @@
+ "app-id" : "org.vikdevelop.VCalculator",
```

```suggestion
{
 "app-id" : "org.vikdevelop.VCalculator",
```

This should make it JSON valid. That's the cause of the build failure.

---

Also, concerning the app-id, `vikdevelop.org` doesn't exist, so you can claim it. `com.github.vikdevelop.VCalculator` would be fine.

---

The invalidity of the json file hasn't be addressed.

---

And what's wrong with it? I've changed the name (and app-id) from org.vikdevelop.VCalculator to com.github.com.vikdevelop.VCalculator, and removed the network socket.

---

You are missing the open curly brace. As I mentionned here https://github.com/flathub/flathub/pull/2365#discussion_r645921804

---

I added sha256 and { to the 1st line.

---

You forgot to add a `,` which make the JSON invalid.

If you need help to use flatpak-builder to try to build locally, let us know.